Modules/FindNetCDF.cmake

```diff
@@ -231,6 +244,9 @@ foreach( _comp IN LISTS _search_components )
       if( NOT _comp MATCHES "^(C)$" )
         target_link_libraries(NetCDF::NetCDF_${_comp} INTERFACE NetCDF::NetCDF_C)
       endif()
+      if(NetCDF_PARALLEL)
+	target_link_libraries(NetCDF::NetCDF_${_comp} INTERFACE MPI::MPI_C)
```

Comment:
Need indent here.
Also, why does the NetCDF_Fortran library need to interface with the MPI_C library?
This should be done before this for-loop and only for the NetCDF_C library.  All other components automatically add `NetCDF_C` to their interface.

Reply:
That makes sense. Admittedly, I didn't really know what INTERFACE was doing. 

You can't do that before the for-loop. The library is created a couple lines above.